app/parse.py

```diff
@@ -1,10 +1,25 @@
 from dataclasses import dataclass
 from enum import Enum
 
+import requests
+from bs4 import BeautifulSoup
+
+HOME_URL = "https://mate.academy/"
+
+# CSS classes
+name_class = ".typography_landingH3__vTjok"
+short_description_class = ".CourseCard_courseDescription__Unsqj"
+course_class = ".CourseCard_cardContainer__7_4lK"
+
 
 class CourseType(Enum):
-    FULL_TIME = "full-time"
-    PART_TIME = "part-time"
+    FULL_TIME = False
+    PART_TIME = True
```

Comment:
are you sure about changing this to True/False? 
In this case in the output you will have something like: `course_type: false`. It's not really readable.